etl/instances_to_filter.py

```diff
@@ -0,0 +1,12 @@
+from enum import Enum
+
+
+class InstancesToFilter(Enum):
+    YEAR = "Q577"
+    CALENDARYEAR = "Q3186692"
+    DISAMBIGUATION = "Q4167410"
+    LIST = "Q13406463"
+
+    @classmethod
+    def list(cls):
+        return list(map(lambda p: p.value, InstancesToFilter))
```

Comment:
Thanks for this change! I like how explicit it reads now.

FYI: in Python it is considered idiomatic to use list comprehension instead of lambdas and map/filter (when possible). Since `Enum` is an iterable, 
the `list()` method is equivalent to  `return [p.value for p in InstancesToFilter]`

Reply:
Done ✅